### PR TITLE
[New Admin API] Pre-Create VNI Ranges (VxLan/GRE)

### PR DESCRIPTION
**Pull request by kevin-zhonghao**
This PR proposes a fix to the issue raised in #371. 

__Root cause__
When customer create a VPC, control plane automatically allocates a VNI for the VPC. The problem is that the VNI range (VxLan range in this case) is created "on the fly" - the range isn't created until the customer allocates some VNI from the range. The logic doesn't sustain well when the customer load is very high at the time of control plane launch.

__Proposed fix__
This PR proposes a new API which allows admin to pre-create VNI ranges at control plane launch time.

### CODE REVIEW

File: services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/entity/NetworkVxlanRange.java [hunk 1]
```diff
@@ -52,8 +52,11 @@ public Long allocateKey() throws Exception {
     }
 
     public void release(Long key) throws Exception {
+//        if (allocated.get(key) == null) {
+//            throw new NetworkKeyAllocNotFoundException();
+//        }
         if (allocated.get(key) == null) {
-            throw new NetworkKeyAllocNotFoundException();
+            return;
```
Comment (xieus) [inline services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/entity/NetworkVxlanRange.java:59]: The question here is: is there any scenario that may need to throw an exception? If the answer is positive or likely, we would need to handle this differently. 

One option is throw NetworkKeyAllocNotFoundException here (no change), and handle the exception differently based on requirement. 
- In this fix, create a new TryReleaseXXX method which catches the exception but don't re-throw, only log info. 
- For other scenarios that requires to see the exception, continue to re-throw.

**Review by xieus — CHANGES_REQUESTED** [review]
@kevin-zhonghao Very prompt fix. Thank you. 

Left one comment for you. Please take a look.

File: services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/entity/NetworkVxlanRange.java (hunk 1 above)
Comment (kevin-zhonghao) [reply services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/entity/NetworkVxlanRange.java:59]: Yeah that's what I think ~ when we could locate the issue, I will modify the code as you suggest

**Review by xieus — APPROVED** [review]
Thanks @kevin-zhonghao I think this PR has identified the root cause and proposed a fundamental fix. Thumbs up 👍

**Comment by xieus** [comment]
@kevin-zhonghao I believe one pending item for this PR to set the access role for this API to be admin only.

Ref: https://github.com/futurewei-cloud/alcor/blob/master/docs/modules/ROOT/pages/high_level/rbac_design.adoc

You likely need to add one entry for the new API: 
https://github.com/futurewei-cloud/alcor/blob/master/services/vpc_manager/src/main/resources/rbac_policy.json

**Pull request merged**